packages/docusaurus-init/templates/facebook/src/pages/index.js

```diff
@@ -52,11 +53,11 @@ function Feature({imageUrl, title, description}) {
   const imgUrl = useBaseUrl(imageUrl);
   return (
     <div className={clsx('col col--4', styles.feature)}>
-      {imgUrl && (
+      <If condition={imgUrl}>
```

Comment:
this is a React anti-pattern and not somthing we'll add

Reply:
Even if you don't want to make code more readable that way, optional chaining should still be used in other places. `features && features.length > 0` is clunky and unnecessary.